### ISSUE TITLE
Pytorch 1.6 Issues with OneShot

### ISSUE THREAD

**Issue by duda1202** [PRBonn/bonnetal#24]
Hi,

I am trying to use the oneshot while using Pytorch 1.6 however it gives me a warning:

UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate.

Do you know how I could solve this? Thank you
